src/ui/public/vislib/visualizations/point_series/column_chart.js

```diff
@@ -105,6 +105,11 @@ export function VislibVisualizationsColumnChartProvider(Private) {
         const groupSpacing = groupWidth * groupSpacingPercentage;
 
         barWidth = (groupWidth - groupSpacing) / groupCount;
+
+        const minBarWidth = 1;
```

Comment:
I don't know how I feel about adding this new `const` right here, just before it's used. It's set to 1, right? It would be hard for people to find what that minimum width is, because it's defined so deep. It seems like it should be a much higher level thing that applies to all `timeScale` bar charts. Thoughts?